api/EXT_blend_minmax.json

```
@@ -13,10 +13,17 @@
           "edge": {
             "version_added": "17"
           },
-          "firefox": {
-            "version_added": "35",
-            "notes": "Before Firefox 47, this extension was not available on Windows."
-          },
+          "firefox": [
+            {
+              "version_added": "47"
+            },
+            {
+              "version_added": "35",
+              "version_removed": "47",
+              "partial_implementation": true,
+              "notes": "Before Firefox 47, this extension was not available on Windows."
```

```suggestion
              "notes": "Not supported on Windows."
```

Since the note applies to the Firefox 35-46 entry, specifying the version in the note doesn't seem that useful, and in fact would cause me to look harder to see if there's a distinction between "Firefox 35-46" and "Before Firefox 47", which there isn't here.